src/utils/form/clipboardUtils.ts

```diff
@@ -0,0 +1,128 @@
+import { tokenValues } from '$stores';
+import { tick } from 'svelte';
+
+export const keydownHandler = async (e: KeyboardEvent, verifyToken: () => void): Promise<void> => {
```

Comment:
Hier schreit ja unsere Pipeline. Weiß jetzt nicht genau wie man das sinnvoll machen kann. Ich hätte die jeweiligen handler der key inputs in extra Funktionen ausgelagert, bzw auch in eine falls das von der Logik möglich ist. Dann hat man keine so große Funktion durch die Funktionsaufrufe im Switch-Case

Reply:
Die Pipeline hat wegen der Reihenfolge der Methoden rumgeschrien, hab das jetzt geändert